src/assets/css/App.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.bg-pink-dark {
+  /* background: rgb(67, 105, 175); */
+  background: #fb5092;
+}
+
+.bg-pink-light {
+  /* background-color: rgb(88, 137, 229) !important; */
+  background-color: #d6608e;
+}
+
+body {
+  /* background-color: rgb(62, 108, 194) !important; */
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.